src/sentry/options/defaults.py

```diff
@@ -167,6 +167,9 @@
 register("store.normalize-after-processing", default=0.0)  # unused
 register("store.disable-trim-in-renormalization", default=0.0)  # unused
 
+# Use nodestore instead of events table for duplicate checking in store
+register("store.use-nodestore", type=Bool, default=True, flags=FLAG_PRIORITIZE_DISK)
```

Comment:
this will be False

Reply:
But the default is True right now?

Reply:
Sorry, it was just a note to myself to flip it later